test/CAS/cas_fs.swift

```diff
@@ -0,0 +1,29 @@
+// RUN: %empty-directory(%t)
+// RUN: mkdir -p %t/empty
+// RUN: mkdir -p %t/cas
+
+// RUN: llvm-cas --cas %t/cas --ingest --data %t/empty > %t/empty.casid
+// RUN: not %target-swift-frontend -typecheck -enable-cas -cas-fs @%t/empty.casid -cas-path %t/cas %s 2>&1 | %FileCheck %s --check-prefix NO-INPUTS
+// NO-INPUTS: error: error opening input file
+
+// RUN: llvm-cas --cas %t/cas --ingest --data %s > %t/source.casid
+// RUN: not %target-swift-frontend -typecheck -enable-cas -cas-fs @%t/source.casid -cas-path %t/cas %s 2>&1 | %FileCheck %s --check-prefix NO-RESOURCES
+// NO-RESOURCES: error: unable to set working directory
+// NO-RESOURCES: error: unable to load standard library
+
+/// Ingest the resource directory to satisfy the file system requirement. Also switch CWD to resource dir.
+// RUN: llvm-cas --cas %t/cas --merge @%t/source.casid %test-resource-dir > %t/full.casid
+// RUN: cd %test-resource-dir
+// RUN: %target-swift-frontend -typecheck -enable-cas -cas-fs @%t/full.casid -cas-path %t/cas %s
+
+/// Try clang importer.
+// RUN: not %target-swift-frontend -typecheck -enable-cas -cas-fs @%t/full.casid -cas-path %t/cas %s -import-objc-header %S/Inputs/objc.h 2>&1 | %FileCheck %s --check-prefix NO-BRIDGING-HEADER
+// NO-BRIDGING-HEADER: error: bridging header
+
+// RUN: llvm-cas --cas %t/cas --merge @%t/full.casid %S/Inputs/objc.h > %t/bridging_header.casid
+// RUN: %target-swift-frontend -typecheck -enable-cas -cas-fs @%t/bridging_header.casid -cas-path %t/cas %s -import-objc-header %S/Inputs/objc.h
+
+/// Clean the CAS to save space.
```

Comment:
What's the issue here? I don't want us to need to clean the cas *after* tests if we can avoid it, since it is hard to maintain.

Reply:
There is no issue. This test ingests entire swift resource directory into CASFS, and depending on your configuration, it can be quite big. In general, we can only ingest the part we need but it is hard to write test here without driver integration.